### ISSUE TITLE
Accessibility menu changes

### ISSUE THREAD

**Issue by wrg20** [web-illinois/illinois_framework_theme#9]
aria-expanded when menu is closed must have value of false. when menu is expanded - value must be true. we have it backwards, and include aria-haspopup=true on each menu that has a child. on mobile - should be able to get to search before menu with tabbing. spacebar and enter key must toggle items.

**Comment by wrg20** [comment]
![image](https://user-images.githubusercontent.com/1544079/107819937-95e34980-6d3f-11eb-855f-5a1c89e91de8.png)

**Comment by wrg20** [comment]
![image](https://user-images.githubusercontent.com/1544079/107985696-2c0dae80-6f90-11eb-8d23-ef862c4989b1.png)
Need to fix some of the accessibility issues found by AXE.  Others pertain to drupal core.
